games-dashboard/games-dashboard.js

```diff
@@ -35,34 +36,85 @@ fetch('https://free-to-play-games-database.p.rapidapi.com/api/games?platform=pc'
   /* SEARCH METHOD WITH KEYUP EVENT START */
   searchInput.addEventListener(`keyup`, (e) =>{
     let searchString = e.target.value.toLowerCase();
-    data.filter((games) => {
-      if (games.title.toLowerCase().includes(searchString) === true) {
+    let results = data.filter((games) => {
+      return games.title.toLowerCase().includes(searchString);
+    });
+
+    container.style.display = `none`;
+    if (results.length === 0) {
+      if (main.lastElementChild.classList.contains(`sara`) || main.lastElementChild.classList.contains(`dahman`)) {
+        main.lastElementChild.remove();
+        let empty = document.createElement(`div`);
+        empty.classList = `message sara`;
+        let emptyPara = document.createElement(`p`);
+        emptyPara.textContent = `No matching results !`;
+        empty.appendChild(emptyPara);
+        main.appendChild(empty);
+      }else {
+        
+        let empty = document.createElement(`div`);
+        empty.classList = `error sara`;
+        let emptyPara = document.createElement(`p`);
+        emptyPara.textContent = `No matching results !`;
+        empty.appendChild(emptyPara);
+        main.appendChild(empty);
+      }
+    }else if (main.lastElementChild.classList.contains(`dahman`) || main.lastElementChild.classList.contains(`sara`)) {
+      main.lastElementChild.remove();
+      let newContainer = document.createElement(`div`);
+      newContainer.addEventListener(`click`, btnClicked);
+      newContainer.classList = `container dahman`;
+      main.appendChild(newContainer);
+      for(let i = 0; i < results.length; i++) {
         let createCard = document.createElement('div');
         createCard.setAttribute('class', 'card');
         let createImg = document.createElement('img');
-        createImg.src = games.thumbnail;
-        createImg.alt = games.title;
+        createImg.src = results[i].thumbnail;
+        createImg.alt = results[i].title;
         let createBtn = document.createElement('button');
         createBtn.setAttribute('index', i);
-        createBtn.setAttribute('id', 'more-btn');
+        createBtn.setAttribute('data-id', results[i].id);
         let createAnchor = document.createElement(`a`);
         createAnchor.href = `../games-details/games-details.html`;
```

Comment:
Make 'createAnchor' the parent of 'createBtn' to enable the whole button to be clicked and lead to the next page.